settingtypes.txt

```diff
@@ -16,6 +16,9 @@ castle_masonry_wood (Wood) bool false
 castle_masonry_ice (Ice) bool false
 castle_masonry_snow (Snow) bool false
 castle_masonry_obsidianbrick (Obsidian Brick) bool false
+castle_masonry_redsandstone (Red Sandstone) bool true
```

Comment:
From a player perspective, I think it would be helpful if there was an indicator which settings that only take an effect with mdl being enabled.

Reply:
done